package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kaliber/elasticsearch",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
Versie pas updaten wanneer je gaat publishen. En als je dan gaat publishen zou ik een major version doen (ondanks dat we denken dat ie backwards compatible is)